src/main/java/com/onfido/qa/webdriver/backend/Chrome.java

```diff
@@ -40,11 +41,11 @@ public MutableCapabilities getOptions(DesiredCapabilities capabilities, Config c
             chromeOptions.addArguments("--use-fake-ui-for-media-stream");
 
             if (!isEmpty(config.fileForFakeAudioCapture)) {
-                chromeOptions.addArguments("--use-file-for-fake-audio-capture=" + getCanonicalPath(config.fileForFakeAudioCapture));
+                chromeOptions.addArguments("--use-file-for-fake-audio-capture=" + getCanonicalPath(filePathForFakeCapture + config.fileForFakeAudioCapture));
```

Comment:
Combining paths should be done with `Paths.get`
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html

Reply:
I've tried like this:
```java
chromeOptions.addArguments("--use-file-for-fake-video-capture=" + getCanonicalPath(Paths.get(filePathForFakeCapture, config.fileForFakeVideoCapture).toString()));
```
And it works fine in macOS but if I run against BrowserStack on Windows where `filePathForFakeCapture=C:\\Users\\hello\\Downloads\\`  we end up with:
```bash
--use-file-for-fake-video-capture=/Users/joao.luz/git/onfido-sdk-ui/test/webtest/C:\\Users\\hello\\Downloads\\/test-stream-passport.mjpeg
```
That is also why I did the regex match to `getPath` instead of `getCanonicalPath` if windows format (L67).
Do you see a way to make this better and still work on Windows?

Reply:
BTW I believe this would work just fine if the code was being executed on Windows. The problem is we are running it on macOS (or Linux on CI) and dealing with Windows paths for BrowserStack.